### DISCUSSION TITLE
Fehler beim laden der Images

### DISCUSSION THREAD

**Discussion by Marvin678** [neuefische/questions#239] [Web] [answered]
Hallo zusammen, 
ich nutze in meiner App die Spoonacular API, um mir verschiedene Küchen anzeigen zu lassen, wo ich dann diverse Rezepte der jeweiligen Küche einsehen kann.
Mein nächster Schritt war, das die Bilder die in der Api sind auch angezeigt werden.
Wenn ich dann eine Küche in mein Suchfeld eingegeben habe und bestätigen will, kommt folgende Fehlermeldung.

<img width="954" alt="Bildschirmfoto 2022-10-11 um 02 48 51" src="https://user-images.githubusercontent.com/108797569/194974018-ed27fb39-ba2a-4ebf-bc14-0062faa6c66e.png">

Wäre super wenn jemand rüber schauen könnte.
https://github.com/Marvin678/capstone-project/pull/6

**Comment by mbosselmann** [comment, marked as answer]
Hallo Marvin,
damit die images der Spoonacular API geladen werden, musst du in deiner Next config Datei erfassen, dass diese URL erlaubt ist. Passe deine `next.config-js` folgendermaßen an und dann sollte es funktionieren.

```js
/** @type {import('next').NextConfig} */
const nextConfig = {
	reactStrictMode: true,
	swcMinify: true,
	compiler: {
		reactRemoveProperties: true,
		styledComponents: true,
	},
	images: {
		domains: ['spoonacular.com'],
	},
};

module.exports = nextConfig;
```
Ich hoffe, dass hilft dir weiter. :)

**Reply by Marvin678** [reply]
Super, hat geholfen. Danke dir :)
